package.json

```diff
@@ -36,11 +36,11 @@
     "lint:md": "remark --frail --quiet .",
     "release": "pnpm --silent build && changeset publish",
     "skuba": "pnpm --silent build && pnpm --silent skuba:exec",
-    "skuba:exec": "node --env-file=.env lib/skuba",
+    "skuba:exec": "node --experimental-vm-modules lib/skuba",
     "stage": "changeset version && node ./.changeset/inject.js && pnpm format",
-    "test": "pnpm --silent skuba test --selectProjects unit",
+    "test": "pnpm --silent skuba test --selectProjects unit --",
```

Comment:
Allows you to do `pnpm test <file>`, same as below. The problem is that --selectProjects accepts varargs